lib/ui/pages/news_list/news_list_page.dart

```diff
@@ -23,41 +24,48 @@ class _NewsPageState extends State<NewsListPage> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
-        child: Padding(
-          padding: EdgeInsets.all(_Constants.pagePadding),
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              PrimaryNewsListItem(
-                news: PrimaryNewsListEntity(
-                  imageUrl: 'https://i.picsum.photos/id/9/800/800.jpg',
-                  title: "New laptops seem to be stuck to the board",
-                  authorName: "Marek Aureliusz",
-                  authorImageUrl: 'https://www.ludoviccareme.com/files/image_211_image_fr.jpg',
-                ),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            PrimaryNewsListItem(
+              news: PrimaryNewsListEntity(
+                imageUrl: 'https://i.picsum.photos/id/9/800/800.jpg',
+                title: "New laptops seem to be stuck to the board",
+                authorName: "Marek Aureliusz",
+                authorImageUrl:
+                    'https://www.ludoviccareme.com/files/image_211_image_fr.jpg',
               ),
-              _buildNewsSectionHeader(),
-              SizedBox(height: _Constants.sectionHeaderPadding),
-              NewsListView(secondaryNewsListEntities: secondaryNewsListEntities),
-            ],
-          ),
+            ),
+            _buildSecondaryNewsSectionHeader(),
+            SizedBox(height: _Constants.sectionHeaderPadding),
+            SecondaryNewsListView(
+                secondaryNewsListEntities: secondaryNewsListEntities),
```

Comment:
a gdyby tak 
```suggestion
                secondaryNewsListEntities: secondaryNewsListEntities.take(3).toList()), 
```
i wyjebać tą listę 3 elementów, a tej drugiej zmienić nazwę. To samo podejście będzie można zrobić dla rzeczywistej listy

Reply:
true, nie wiem czemu zrobiłem drugiego mocka :D